recipe/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+mkdir ${PREFIX}/include
```

Comment:
```suggestion
if [ -d ${PREFIX}/include ];
then
    echo "Directory ${PREFIX}/include exists"
else
    echo "Error: Directory does not exists"
    mkdir ${PREFIX}/include
fi
```
This will fix an issue on `osx`